scripts/build_utils.sh

```diff
@@ -256,40 +256,27 @@ get_rpm_dist() {
     # creates a DISTRO_VERSION and DISTRO global variable for
     # use in constructing chacra urls for rpm distros
 
-    LSB_RELEASE=/usr/bin/lsb_release
-    [ ! -x $LSB_RELEASE ] && echo unknown && exit
+    source /etc/os-release
 
-    ID=`$LSB_RELEASE --short --id`
+    RELEASE=$VERSION_ID
+    DISTRO=$ID
 
     case $ID in
-    RedHatEnterpriseServer)
-        RELEASE=`$LSB_RELEASE --short --release | cut -d. -f1`
+    rhel)
         DIST=rhel$RELEASE
-        DISTRO=rhel
         ;;
-    CentOS|CentOSStream)
-        RELEASE=`$LSB_RELEASE --short --release | cut -d. -f1`
+    centos)
         DIST=el$RELEASE
-        DISTRO=centos
         ;;
-    Fedora)
-        RELEASE=`$LSB_RELEASE --short --release`
+    fedora)
         DIST=fc$RELEASE
-        DISTRO=fedora
         ;;
-    SUSE\ LINUX|openSUSE)
-        DESC=`$LSB_RELEASE --short --description`
-        RELEASE=`$LSB_RELEASE --short --release`
-        case $DESC in
-        *openSUSE*)
-                DIST=opensuse$RELEASE
-                DISTRO=opensuse
-            ;;
-        *Enterprise*)
-                DIST=sles$RELEASE
-                DISTRO=sles
-                ;;
-            esac
+    sles)
+        DIST=sles$RELEASE
+        ;;
+    opensuse)
```

Comment:
I pulled `registry.opensuse.org/opensuse/leap:latest`, and here's `/etc/os-release`:

```
cat /etc/os-release 
NAME="openSUSE Leap"
VERSION="15.3"
ID="opensuse-leap"
ID_LIKE="suse opensuse"
VERSION_ID="15.3"
PRETTY_NAME="openSUSE Leap 15.3"
ANSI_COLOR="0;32"
CPE_NAME="cpe:/o:opensuse:leap:15.3"
BUG_REPORT_URL="https://bugs.opensuse.org"
HOME_URL="https://www.opensuse.org/"
```

```suggestion
    opensuse-leap)
```